fragments/megamenu/index.yml

```diff
@@ -0,0 +1,325 @@
+        - text: Get Veteran ID Cards
+          href: https://www.va.gov/records/get-veteran-id-cards/
+      columnTwo:
+        title: VA
+        links:
+        - text: Request Your Military Records (DD214)
+          href: https://www.va.gov/records/get-military-service-records/
+        - text: How to Apply for a Discharge Upgrade
+          href: https://www.va.gov/discharge-upgrade-instructions
+        - text: View Your VA Payment History
+          href: https://www.va.gov/va-payment-history/
+        - text: Search Historical Military Records (National Archives)
+          href: https://www.archives.gov/
+          target: _blank
+      fragments:
+        columnThree: promotions/records
+      seeAllLink:
+        text: Records
+        href: https://www.va.gov/records/
+- title: About VA
+  menuSections:
+    mainColumn:
+      title: VA Organizations
+      links:
+      - text: Veterans Health Administration
+        href: https://www.va.gov/health/
+      - text: Veterans Benefits Administration
+        href: https://www.benefits.va.gov/benefits/
+      - text: National Cemetery Administration
+        href: https://www.cem.va.gov
+      - text: VA Leadership
+        href: https://www.va.gov/opa/bios/index.asp
+      - text: Public Affairs
+        href: https://www.va.gov/OPA/index.asp
+      - text: Congressional Affairs
+        href: https://www.va.gov/oca/index.asp
+      - text: All VA Offices and Organizations
+        href: https://www.va.gov/landing_organizations.htm
+    columnOne:
+      title: Innovation at VA
+      links:
+      - text: Health Research
+        href: https://www.research.va.gov
+      - text: Public Health
+        href: https://www.publichealth.va.gov
+      - text: Veterans Choice Program
+        href: https://www.va.gov/COMMUNITYCARE/programs/veterans/VCP/index.asp
+      - text: VA Open Data
+        href: https://www.data.va.gov
+        target: _blank
+      - text: Veterans Analysis and Statistics
+        href: https://www.va.gov/VETDATA/index.asp
+      - text: Appeals Modernization
+        href: https://www.benefits.va.gov/benefits/appeals.asp
+      - text: VA Center for Innovation
+        href: https://www.innovation.va.gov
+      - text: Recovery Act
+        href: https://www.va.gov/recovery/
+    columnTwo:
+      title: Learn About VA
+      links:
+      - text: VA Mission, Vision, and Values
+        href: https://www.va.gov/ABOUT_VA/index.asp
+      - text: History of VA
+        href: https://www.va.gov/about_va/vahistory.asp
+      - text: VA Plans, Budget, and Performance
+        href: https://www.va.gov/performance/
+      - text: National Cemetery History Program
+        href: https://www.cem.va.gov/cem/history/index.asp
+      - text: Veterans Legacy Program
+        href: https://www.cem.va.gov/cem/legacy/index.asp
+      - text: Volunteer or Donate
+        href: https://www.volunteer.va.gov/index.asp
+    fragments:
+        columnThree: promotions/about-va
```

Comment:
Is there a way to tie this to a meta data in the fragment instead of to its file name? (maybe using the file name is idiomatic- I don't have a lot of experience with using metalsmith / yml)